python/dlr/api.py

```diff
@@ -26,27 +26,41 @@ def get_output_names(self):
     def run(self, input_data):
         raise NotImplementedError
 
+
+def _find_model_file(model_path, ext):
+    if os.path.isfile(model_path) and model_path.endswith(ext):
+        return model_path
+    model_files = glob.glob(os.path.abspath(os.path.join(model_path, ext)))
+    if len(model_files) > 1:
+        raise ValueError('Multiple {} files found under {}'.format(ext, mdel_path))
+    elif len(model_files) == 1:
+        return model_files[0]
+    return None
+
+
 # Wrapper class
 class DLRModel(IDLRModel):
-    def __init__(self, model_path, dev_type='cpu', dev_id=0):
-        #Determine if 3rdparty package needed
-        tf_model_path = None
-        if os.path.isfile(model_path):    
-            if model_path.endswith('.pb'):
-                tf_model_path = model_path
-        else:
-            model_files = glob.glob(os.path.abspath(os.path.join(model_path, '*.pb')))
-            if len(model_files) > 1:
-                raise ValueError('Multiple .pb files found under ' + model_path)
-            elif len(model_files) == 1:
-                tf_model_path = model_files[0]
-        # Default to DLR model
+    def __init__(self, model_path, dev_type=None, dev_id=None):
+        # Find correct runtime implementation for the model
+        # Tensorflow
+        tf_model_path = _find_model_file(model_path, '.pb')
         if tf_model_path is not None:
             from .tf_model import TFModelImpl
             self._impl = TFModelImpl(tf_model_path, dev_type, dev_id)
-        else:
-            from .dlr_model import DLRModelImpl
-            self._impl = DLRModelImpl(model_path, dev_type, dev_id) 
+            return
+        # TFLite
```

Comment:
if we find both pb and tflite, warn user which one is used or just raise error?

Reply:
fixed. will throw the Exception if both found

Reply:
I am okay with raising an error, but I think warning (then using TF Lite by default) might be a better option.

Reply:
We agreed to raise the error here